enhancements/console/dynamic-plugins.md

```diff
@@ -445,7 +445,7 @@ metadata:
 spec:
   displayName: 'Advanced Cluster Management'
   i18n:
-    enabled: true
+    loadType: Preload
```

Comment:
What are the possible values for `loadType`?

Reply:
@spadgett possible types are `Preload` & `Lazy`

Reply:
I don't think we need `Lazy` since there is nothing the console needs to do. And what if my plugin doesn't have translations? Should the other option be `None`?

It would be good to see what other enums look like for properties that are basically enable/disable.

Reply:
> I don't think we need Lazy since there is nothing the console needs to do. And what if my plugin doesn't have translations? 
@spadgett I added it since it would better describe the way how loading types that we support. `Preload` will preload the locales. `Lazy` wont preload, only wait till plugin will lazy load them, so its kinda a superset to `None`